lbz/request.py

```diff
@@ -33,21 +33,21 @@ def __init__(
         query_params: dict = None,
         user: User = None,
     ):
-        self.query_params = MultiDict(query_params)
-        self.headers = headers
-        self.uri_params = uri_params
-        self.method = method
-        self.context = context
+        self.query_params: MultiDict = MultiDict(query_params or {})
+        self.headers: CIMultiDict = headers
+        self.uri_params: dict = uri_params
+        self.method: str = method
+        self.context: dict = context
         self.stage_vars = stage_vars
-        self.user = user
-        self._is_base64_encoded = is_base64_encoded
-        self._body = body
+        self.user: Optional[User] = user
+        self._is_base64_encoded: bool = is_base64_encoded
+        self._body: Union[str, bytes] = body
```

Comment:
In general, all of those attributes rely on the parameters given into `__init__`, so you don't have to specify types here. But if something is incompatible it should be changed there (as a type of input value), not here 😉